api/app.go

```diff
@@ -125,27 +117,29 @@ func (a App) healthCheckHandler(w http.ResponseWriter, r *http.Request) {
 }
 
 func (a *App) initializeRoutes() {
-	a.Router.HandleFunc("/parcelle/{idu:"+iduRegex+"}", a.getById).Methods("GET")
+	a.Router.PathPrefix("/viewer/").Handler(http.StripPrefix("/viewer/", http.FileServer(http.Dir("./views")))).Methods("GET")
```

Comment:
Viewer is a good idea. However best would to remove viewer for this PR and address it in another PR. Generally speaking we think the viewer should be optional.

Reply:
The main goal of the project is to provide a basic and simple prototype to use our data. It would become incredibly complex if the user who want to test it must clone multiple projects and link them together in order to get it work as he need.
We'll keep the code within the same project, and add the `VIEWER` boolean environment variable to control whether it should load the feature or not at startup.